praw/models/reddit/modmail.py

```diff
@@ -65,11 +65,17 @@ def parse(cls, data, reddit):
 
         return cls(reddit, _data=conversation)
 
-    def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
-                 _data=None):
-        """Construct an instance of the ModmailConversation object."""
-        super(ModmailConversation, self).__init__(reddit, _data)
+    def __init__(self, reddit, mark_read=False,
```

Comment:
Could we keep `id` before `mark_read` here please? And use them in the same order elsewhere?